flow_action_components/CollectionProcessors/force-app/main/default/classes/SortCollection.cls

```diff
@@ -21,7 +21,7 @@ global with sharing class SortCollection {
             }
 
             if (!outputCollection.isEmpty()) {
-                Map<String, String> fieldTypes = CollectionProcessorsUtils.getFieldTypes(outputCollection[0], allFields);
+                Map<String, String> fieldTypes = getFieldTypes(outputCollection[0], allFields);
```

Comment:
Here, this action was using a single shared getFieldTypes, which provides some reuse. What was the reason you decided to create a copy of the code here?

Reply:
I don't recall working on that section of code - I'm thinking it was from an older version of SortCollection.

Reply:
let's stick with the version that's in ColectionProcessorsUtils in that case.